tests/test_yaml.py

```diff
@@ -18,3 +19,36 @@ def test_validate():
     # should be well-formed
     file_ = "doc/example/example_Fujita/Fujita.yaml"
     validate(file_)
+
+
+def test_create_problem_yaml():
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Missing function or method docstring](https://app.codacy.com/manual/dweindl/PEtab/pullRequest?prid=5219106)

Reply:
this is not done for tests, right?

Reply:
can be ignored, yeah